.idea/MITR-Project.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
could you remove the .idea folder from the branch? not sure how to do it exactly with desktop, but will require u to add .idea into the gitignore and remove the .idea folder from the repository. (run the following command in the root folder of MITR-Project to remove .idea from the branch) `git rm --cached  -r .idea`.